mmv1/templates/terraform/examples/volume_basic.tf.erb

```diff
@@ -13,6 +13,7 @@ resource "google_netapp_volume" "<%= ctx[:primary_resource_id] %>" {
   share_name = "<%= ctx[:vars]['volume_name'] %>"
   storage_pool = google_netapp_storage_pool.default.name
   protocols = ["NFSV3"]
+  deletion_policy = "DEFAULT"
```

Comment:
I added it intentionally so the user is notified to do a deliberate decision on it's value when copy/pasting the example. IMHO users want to set it to FORCE most of the time to make TF work as expected (= deleting a volume on destroy). This would suggest to make FORCE the default value.
On the other side, we have seen users struggling to understand the relationship between volume and snapshots (snapshots are nested resources that go away if the volume goes away). They deleted the volume and tried to create a new one from a snapshot, which was gone by then. They thought snapshots work like GCE snapshots, which are independent copies. They aren't.

Having said that: While I think that users want to use FORCE most of the time, I always recommend a "better save than sorry" policy when it comes to storage and advocate the make DEFAULT the - well - default and FORCE and intentional decision. But i want to raise awareness that this is a decision the user should consider. Therefore, I added it explicitly to the basic example.

Given that background, what's your take?

Reply:
I see. So the purpose of it in is to expose it in the doc. Yeah, that's valid point!